src/Compilers/CSharp/Test/Syntax/Parsing/ReplaceParsingTests.cs

```diff
@@ -100,5 +95,13 @@ private void OriginalInMember(string text, bool inReplace)
             var expectedKind = inReplace ? SyntaxKind.OriginalExpression : SyntaxKind.IdentifierName;
             Assert.Equal(expectedKind, expr.Kind());
         }
+
+        [Fact]
+        [WorkItem(11503, "https://github.com/dotnet/roslyn/issues/11503")]
+        public void MethodCalledReplace()
+        {
+            var root = SyntaxFactory.ParseCompilationUnit("class C { public static object replace() { } }", options: TestOptions.Regular.WithReplaceFeature());
+            root.Errors().Verify();
```

Comment:
Consider adding end to end test. Verify that binding works, the method can be called, etc.